src/compiler/utilities.ts

```diff
@@ -2083,6 +2083,12 @@ namespace ts {
         return start <= textSpanEnd(span) && end >= span.start;
     }
 
+    export function textSpanIntersectsWith2(start1: number, length1: number, start2: number, length2: number) {

```

Comment:
"numbers do not belong in API names", I will make sure i put that on my tombstone :D